bindings/core/tests/secrets_debug.rs

```diff
@@ -40,7 +40,7 @@ fn method_interface_secrets_debug() {
     };
     assert_eq!(format!("{:?}", wallet_method), "VerifyMnemonic { mnemonic: <omitted> }");
 
-    let response = Response::GeneratedMnemonic("mnemonic".to_string());
+    let response = Response::GeneratedMnemonic("mnemonic".to_owned().to_mnemonic().unwrap());
```

Comment:
This needs to be a valid mnemonic now

Reply:
of course ... I guess I go with the awesome "giant dynamic museum toddler"!!